GatewayMicroService/GatewayMicroService/source/api/login_api.py

```diff
@@ -0,0 +1,16 @@
+from fastapi import APIRouter, Request
+import httpx
+router = APIRouter()
+
+
+
+@router.get("/{info}")
+async def get_user(user_id:str):
+    return {"message": "the role of this service is to login user "}
+
+@router.get("/forward")
+async def forward_request_parser(request: Request):
+    url = "http://microservice-url:8000"  # Replace with the URL of your microservice
```

Comment:
add it into configuration class and get it from there. also make it configurable